ibm/service/vpc/data_source_ibm_is_bare_metal_server.go

```diff
@@ -558,6 +558,19 @@ func dataSourceIBMISBareMetalServerRead(context context.Context, d *schema.Resou
 				currentPrimNic[isInstanceNicAllowIPSpoofing] = *primNic.AllowIPSpoofing
 				currentPrimNic[isBareMetalServerNicPortSpeed] = *primNic.PortSpeed
 
+				if len(primNic.SecurityGroups) != 0 {
```

Comment:
Please update the documentation and examples for all the datasources and resources accordingly..